code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -270,6 +270,10 @@
 
 	msg += "</span>"
 
+	if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && !appears_dead && (src != user) && src.mind.isholy)
+		msg += "[t_He] [t_has] a holy aura about [t_him].\n"
+		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "religious_comfort", /datum/mood_event/religiously_comforted)
+
 	if(!appears_dead)

```

Comment:
you could prob else this once you do that

Reply:
What am I else'ing this against? Wouldn't it just be an independent check?

Reply:
who knows this was 9 hours ago lol